usr/share/gamescope-session-plus/sessions.d/playtron

```diff
@@ -26,8 +26,17 @@ if lspci | grep -i vga | grep -i nvidia > /dev/null; then
 	# required for Nvidia GPUs to work with webview
 	# breaks webview for AMD and Intel GPUs
 	WEBKIT_DISABLE_DMABUF_RENDERER=1
+	mkdir -p "${HOME}"/.config/gamescope/scripts/ 2> /dev/null
```

Comment:
I found a much cleaner way. We can check an environment variable in the lua script and conditionally disable explicit sync based on it.

I have tested and added this script into the gamescope session with this commit:
https://github.com/ChimeraOS/gamescope-session/commit/57c46d19724ca2766e81cd2f5ee96b3baf3b8ae3

So here, all we need to do is add the line `GAMESCOPE_DISABLE_EXPLICIT_SYNC=1`